apps/remix-ide/src/app/editor/editor.js

```diff
@@ -189,14 +189,17 @@ class Editor extends Plugin {
     this.editor.on('changeSession', () => {
       this._onChange()
       this.event.trigger('sessionSwitched', [])
-
       this.editor.getSession().on('change', () => {
         this._onChange()
         this.event.trigger('contentChanged', [])
       })
     })
   }
 
+  onActivation () {
```

Comment:
might need to do something like `onDeactivation` : `this.off('sidePanel', 'focusChanged')` .
Please check the syntax.

Reply:
right